pkgs/applications/version-management/sr-ht/todo.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchgit
+, srht, python3Packages }:
+
+with python3Packages;
+
+buildPythonPackage rec {
+  name = "todo-sr-ht-${version}";
+  version = "0.30.5";
+
+  src = fetchgit {
+    url = "https://git.sr.ht/~sircmpwn/todo.sr.ht";
+    rev = version;
+    sha256 = "197bwhqjg1bkk825420813lqb1ycpn79qja95hpvwwcif396dhn9";
+  };
+
+  patches = [
+    ./env.patch
```

Comment:
Why's this patch needed?

Reply:
I can't seem to get the sitepackages for srht to show unless I explicitly specify `SRHT_PATH` but for it to get to `SRHT_PATH` I need to get rid of the first part in the if statement. https://git.sr.ht/~sircmpwn/git.sr.ht/tree/master/setup.py#L8

PYTHONPATH seems to be fine in the derivation, but `getsitepackages` only returns the python derivation's sitepackages.